ports/uni-algo/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "uni-algo",
+  "version-semver": "0.7.0",
+  "description": "Unicode Algorithms Implementation",
+  "homepage": "https://github.com/uni-algo/uni-algo",
+  "license": "Unlicense OR MIT",
```

```suggestion
  "license": "Unlicense OR MIT",
```
If there is no license, the value is `null`.

---

I'm not sure what you meant. The license matches what upstream says.

---

Upstream `Public Domain License` means to declare the license of the current library, and to be under the license of `MIT`.
The license should be `MIT`.

---

AFAICS the Public Domain License is correctly identified as `Unlicense`.  
https://tools.spdx.org/app/check_license/ 
So `"Unlicense OR MIT"` should be correct.